Source/SDFutureExtensions/Private/FutureExtensionsStaticFuncs.cpp

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include "FutureExtensions.h"

```

Comment:
This introduces a compile error based on the order of compilation. 
It expects the corresponding header to be the first include, but the plugin is written to assume the include order defined in FutureExtensions.h.

Any thoughts on how to solve this.